extensions/3DTILES_metadata/README.md

```diff
@@ -0,0 +1,256 @@
+# 3DTILES_metadata Extension
```

Comment:
Is this document still a work-in-progress? I see examples with namespaces but no documentation explaining it.

Reply:
Examples have been updated to not use namespaces or enums.